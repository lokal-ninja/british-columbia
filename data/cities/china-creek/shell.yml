id: 545071738
type: way
latitude: 49.22166
longitude: -117.69269
postcode: "V0G 1G0"
city: China Creek
street: "China Creek Road"
housenumber: 970
phone: ""
opening_hours: ""
website: ""
