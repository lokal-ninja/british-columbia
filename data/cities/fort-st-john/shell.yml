id: 159801247
type: way
latitude: 56.236146100000006
longitude: -120.8409402
postcode: "V1J 6E2"
city: Fort St John
street: "96A Street"
housenumber: 8908
phone: ""
opening_hours: ""
website: ""
