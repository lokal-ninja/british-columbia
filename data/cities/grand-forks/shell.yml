id: 545311617
type: way
latitude: 49.02953
longitude: -118.45324
postcode: "V0H 1H0"
city: Grand Forks
street: "Central Avenue"
housenumber: 1512
phone: "+1-250-442-5788"
opening_hours: ""
website: ""
