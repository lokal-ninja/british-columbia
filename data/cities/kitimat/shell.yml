id: 195170592
type: way
latitude: 54.05814
longitude: -128.64348
postcode: "V8C 1E8"
city: Kitimat
street: "North Lahakas Boulevard"
housenumber: 1065
phone: "+1-250-639-9288"
opening_hours: ""
website: ""