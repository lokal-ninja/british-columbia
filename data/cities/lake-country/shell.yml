id: 456146698
type: way
latitude: 50.02014
longitude: -119.40576
postcode: ""
city: Lake Country
street: ""
housenumber: 
phone: ""
opening_hours: ""
website: ""