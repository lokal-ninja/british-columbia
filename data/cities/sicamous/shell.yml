id: 545331865
type: node
latitude: 50.837
longitude: -118.97633
postcode: V0E 2V0
city: Sicamous
street: Eagle Pass Way
housenumber: "1229"
phone: +1-250-836-2822
opening_hours: ""
website: ""
