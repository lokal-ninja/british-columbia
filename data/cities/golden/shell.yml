id: 144459616
type: way
latitude: 51.31533
longitude: -116.9738
postcode: "V0A 1H0"
city: Golden
street: "Trans-Canada Highway"
housenumber: 1319
phone: "+1-250-344-6613"
opening_hours: ""
website: ""
